### PR TITLE
Fixed grammar, phrasing, and Title Case mistakes in en_us.json

### PR DESCRIPTION
**Pull request by benserwa**

### CODE REVIEW

File: src/main/resources/assets/timetravelmod/lang/en_us.json [hunk 1]
```diff
@@ -9,50 +9,50 @@
...
-    "gui.tmengineerbook.tmstats": "Max tier: %s | Engine cooldown time: %s seconds",
-    "gui.tmengineerbook.howto": "How to build it",
+    "gui.tmengineerbook.tmsintroduction": "To travel to other Time Lines you will need a Time Machine. A Time Machine is a multi-block structure made with four different blocks: basic blocks (the structure), core blocks (the engine), upgrade blocks (these change behaviour), and a control panel. A Time Machine may have more than one valid block for any of the previous categories. Every block must be placed correctly, although Time Machine Upgrades can optionally replace any basic blocks in the structure. After you've built your Time Machine, go inside it and right-click the control panel. A menu will allow you to select your destination Time Line. The available Time Lines will depend on the Time Machine tier. Select one and you will Time Travel for the first time. The Time Machine's engine needs a moment to cool down before it can be used again. If you try to mine the engine while it's cooling down, it will explode, leaving a time hole. There is also a very low chance of a Temporal Explosion occurring any time you use a Time Machine.",
```
Comment (naritanara) [inline src/main/resources/assets/timetravelmod/lang/en_us.json]: > although Time Machine Upgrades can optionally replace any basic blocks in the structure

TM upgrades always replace basic blocks, this line is a little bit missleading

**Review by naritanara — CHANGES_REQUESTED** [review]
Thank's for your help with the English translation! I marked some things that I would prefer to change before merging.

File: src/main/resources/assets/timetravelmod/lang/en_us.json [hunk 2]
```diff
@@ -72,10 +72,10 @@
     "timetravelmod.generic.and" : "%s and %s",
     "tm.timetravelmod.creative.description": "A special Time Machine. This Time Machine is only available in creative mode and allows to time travel without building anything. Instead, you only have to use the item and you will have access to all Time Lines. The Creative Time Machine also doesn't have any cool down time.",
     "tm.timetravelmod.creative.name": "Creative time machine",
-    "tm.timetravelmod.tier1.description": "A very simple Time Machine. You will only reach the Old West.",
+    "tm.timetravelmod.tier1.description": "A very simple Time Machine. You can only visit the Old West.",
```
Comment (naritanara) [inline src/main/resources/assets/timetravelmod/lang/en_us.json]: Why is this changed? I personally prefer the original one.

**Review by naritanara — CHANGES_REQUESTED** [review]

**Comment by benserwa** [comment]
Found some more stuff and will work on these. Can I recommend that "your Time Line" be changed to "the Present"? I think that's a more intuitively explanatory term to use there.

**Comment by benserwa** [comment]
I also want to change the temporal error death message to something cooler but it's a matter of taste. I wanted to do the Chrono Trigger reference with "But %s's future refused to change..." but I'm not sure that gets the point across.

**Comment by benserwa** [comment]
I was a little confused about the meaning of the "tmsintroduction" instructions. Is it accurate now?

**Review by naritanara — APPROVED** [review]
Looks good to me!

**Comment by naritanara** [comment]
> Found some more stuff and will work on these. Can I recommend that "your Time Line" be changed to "the Present"? I think that's a more intuitively explanatory term to use there.

Make a PR with this, as I don't understand quite well what do you mean.

**Comment by naritanara** [comment]
> I also want to change the temporal error death message to something cooler but it's a matter of taste. I wanted to do the Chrono Trigger reference with "But %s's future refused to change..." but I'm not sure that gets the point across.

Make a PR too and I will check.

**Pull request merged**